src/games/superstitious-counting/main.tsx

```diff
@@ -1,23 +1,27 @@
-import { Client } from 'boardgame.io/react';
-import { Local } from 'boardgame.io/multiplayer';
+import { MyClient } from '../../common/myclient';
 import { MyGame } from './game';
 import { MyBoard } from './board';
-import MyBot from './mybot';
+import { strategy } from './strategy';
 
-const SuperstitiousCounting = Client({
-  game: MyGame,
-  board: MyBoard,
-  multiplayer: Local(
-    {
-      bots: { '0': MyBot }
-    }
-  ),
-});
+let description = <p className="text-justify">
```

Comment:
why isn't text justify styling the responsibility of board wrapper, and here the argument would be a plain string?

Reply:
I think the plan is to have formatted descriptions (pictures, etc.).

It could be better to use not raw text (`inline` mode, like `<i><b>`, etc.), but `block` mode (`<div><p>`, etc.) to create better robust formatting.

This is just me saying a point that may or may not have been the intent of the author, I have not messed with this code :)